### PR TITLE
V15

### PR DESCRIPTION
**Pull request by danijeel**
initial support for client sdk v15, I will do a follow up PR with additional acceptance test and updated documentation for new attributes in 5.4, to reduce the size of the PRs.

acceptance test for `5.4.1-24694`
<details>

```
make testacc
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./appgate -v -count 1 -parallel 20  -timeout 120m
=== RUN   TestNewClient
--- PASS: TestNewClient (0.00s)
=== RUN   TestLoginInternalServerError
--- PASS: TestLoginInternalServerError (0.03s)
=== RUN   TestConfig
=== RUN   TestConfig/test_before_5.4
=== RUN   TestConfig/test_5.4_login
=== RUN   TestConfig/invalid_client_version
--- PASS: TestConfig (0.01s)
    --- PASS: TestConfig/test_before_5.4 (0.00s)
    --- PASS: TestConfig/test_5.4_login (0.00s)
    --- PASS: TestConfig/invalid_client_version (0.00s)
=== RUN   TestAccAppgateAdministrativeRoleDataSource
=== PAUSE TestAccAppgateAdministrativeRoleDataSource
=== RUN   TestAccAppgateApplianceCustomizationDataSource
=== PAUSE TestAccAppgateApplianceCustomizationDataSource
=== RUN   TestAccAppgateApplianceSeedDataSource
--- PASS: TestAccAppgateApplianceSeedDataSource (11.64s)
=== RUN   TestAccAppgateCADataSource
--- PASS: TestAccAppgateCADataSource (4.10s)
=== RUN   TestAccAppgateGlobalSettingsDataSource
--- PASS: TestAccAppgateGlobalSettingsDataSource (9.31s)
=== RUN   TestAccAppgateIdentityProviderDataSource
--- PASS: TestAccAppgateIdentityProviderDataSource (4.38s)
=== RUN   TestAccAppgateIPPoolDataSource
--- PASS: TestAccAppgateIPPoolDataSource (4.35s)
=== RUN   TestAccAppgateLocalUserDataSource
--- PASS: TestAccAppgateLocalUserDataSource (5.91s)
=== RUN   TestAccAppgateMfaProviderDataSource
=== PAUSE TestAccAppgateMfaProviderDataSource
=== RUN   TestAccAppgateTrustedCertificateDataSource
=== PAUSE TestAccAppgateTrustedCertificateDataSource
=== RUN   TestProvider
--- PASS: TestProvider (0.00s)
=== RUN   TestProvider_impl
--- PASS: TestProvider_impl (0.00s)
=== RUN   TestAccadministrativeRoleBasic
=== PAUSE TestAccadministrativeRoleBasic
=== RUN   TestAccadministrativeRoleWithScope
=== PAUSE TestAccadministrativeRoleWithScope
=== RUN   TestAccadministrativeMultiplePrivilegesValidation
=== PAUSE TestAccadministrativeMultiplePrivilegesValidation
=== RUN   TestAccadministrativeMultiplePrivilegesValidation52
=== PAUSE TestAccadministrativeMultiplePrivilegesValidation52
=== RUN   TestAccApplianceCustomizationBasic
=== PAUSE TestAccApplianceCustomizationBasic
=== RUN   TestAccApplianceBasicController
=== PAUSE TestAccApplianceBasicController
=== RUN   TestAccApplianceConnector
=== PAUSE TestAccApplianceConnector
=== RUN   TestAccApplianceBasicGateway
=== PAUSE TestAccApplianceBasicGateway
=== RUN   TestAccApplianceBasicControllerWithoutOverrideSPA
=== PAUSE TestAccApplianceBasicControllerWithoutOverrideSPA
=== RUN   TestAccApplianceBasicControllerOverriderSPADisabled
=== PAUSE TestAccApplianceBasicControllerOverriderSPADisabled
=== RUN   TestAccBlacklistUserBasic
=== PAUSE TestAccBlacklistUserBasic
=== RUN   TestAccClientConnectionsBasic
--- PASS: TestAccClientConnectionsBasic (5.00s)
=== RUN   TestAccClientProfileBasic
=== PAUSE TestAccClientProfileBasic
=== RUN   TestAccConditionBasic
=== PAUSE TestAccConditionBasic
=== RUN   TestAccCriteriaScriptBasic
=== PAUSE TestAccCriteriaScriptBasic
=== RUN   TestAccDeviceScriptBasic
=== PAUSE TestAccDeviceScriptBasic
=== RUN   TestAccEntitlementScriptBasic
=== PAUSE TestAccEntitlementScriptBasic
=== RUN   TestAccEntitlementBasicPing
=== PAUSE TestAccEntitlementBasicPing
=== RUN   TestAccEntitlementBasicWithMonitor
=== PAUSE TestAccEntitlementBasicWithMonitor
=== RUN   TestAccEntitlementUpdateActionOrder
=== PAUSE TestAccEntitlementUpdateActionOrder
=== RUN   TestAccEntitlementUpdateActionHostOrder
=== PAUSE TestAccEntitlementUpdateActionHostOrder
=== RUN   TestAccEntitlementUpdateActionPortsSetOrder
=== PAUSE TestAccEntitlementUpdateActionPortsSetOrder
=== RUN   TestAccGlobalSettingsBasic
=== PAUSE TestAccGlobalSettingsBasic
=== RUN   TestAccConnectorIdentityProviderBasic
--- PASS: TestAccConnectorIdentityProviderBasic (7.89s)
=== RUN   TestAccLdapCertificateIdentityProvidervBasic
=== PAUSE TestAccLdapCertificateIdentityProvidervBasic
=== RUN   TestAccLdapIdentityProviderBasic
=== PAUSE TestAccLdapIdentityProviderBasic
=== RUN   TestAccLocalDatabaseIdentityProviderBasic
--- PASS: TestAccLocalDatabaseIdentityProviderBasic (14.75s)
=== RUN   TestAccRadiusIdentityProviderBasic
=== PAUSE TestAccRadiusIdentityProviderBasic
=== RUN   TestAccSamlIdentityProviderBasic
=== PAUSE TestAccSamlIdentityProviderBasic
=== RUN   TestAccIPPoolBasic
=== PAUSE TestAccIPPoolBasic
=== RUN   TestAccLocalUserBasic
=== PAUSE TestAccLocalUserBasic
=== RUN   TestAccMfaProviderBasic
=== PAUSE TestAccMfaProviderBasic
=== RUN   TestAccAdminMfaSettingsBasic
=== PAUSE TestAccAdminMfaSettingsBasic
=== RUN   TestAccPolicyBasic
=== PAUSE TestAccPolicyBasic
=== RUN   TestAccRingfenceRuleBasicICMP
=== PAUSE TestAccRingfenceRuleBasicICMP
=== RUN   TestAccRingfenceRuleBasicTCP
=== PAUSE TestAccRingfenceRuleBasicTCP
=== RUN   TestAccSiteBasic
=== PAUSE TestAccSiteBasic
=== RUN   TestAccSiteBasicAwsResolverWithoutSecret
=== PAUSE TestAccSiteBasicAwsResolverWithoutSecret
=== RUN   TestAccTrustedCertificateBasic
=== PAUSE TestAccTrustedCertificateBasic
=== CONT  TestAccAppgateAdministrativeRoleDataSource
=== CONT  TestAccEntitlementBasicWithMonitor
=== CONT  TestAccLocalUserBasic
=== CONT  TestAccTrustedCertificateBasic
=== CONT  TestAccSiteBasicAwsResolverWithoutSecret
=== CONT  TestAccSiteBasic
=== CONT  TestAccRingfenceRuleBasicTCP
=== CONT  TestAccRingfenceRuleBasicICMP
=== CONT  TestAccPolicyBasic
=== CONT  TestAccAdminMfaSettingsBasic
=== CONT  TestAccMfaProviderBasic
=== CONT  TestAccLdapCertificateIdentityProvidervBasic
=== CONT  TestAccIPPoolBasic
=== CONT  TestAccSamlIdentityProviderBasic
=== CONT  TestAccRadiusIdentityProviderBasic
=== CONT  TestAccLdapIdentityProviderBasic
=== CONT  TestAccApplianceBasicGateway
=== CONT  TestAccEntitlementBasicPing
=== CONT  TestAccEntitlementScriptBasic
=== CONT  TestAccDeviceScriptBasic
--- PASS: TestAccAppgateAdministrativeRoleDataSource (19.38s)
=== CONT  TestAccCriteriaScriptBasic
=== CONT  TestAccConditionBasic
--- PASS: TestAccRingfenceRuleBasicICMP (22.45s)
--- PASS: TestAccAdminMfaSettingsBasic (22.56s)
=== CONT  TestAccClientProfileBasic
--- PASS: TestAccPolicyBasic (22.60s)
=== CONT  TestAccApplianceConnector
--- PASS: TestAccLocalUserBasic (23.11s)
=== CONT  TestAccBlacklistUserBasic
--- PASS: TestAccEntitlementScriptBasic (23.21s)
=== CONT  TestAccApplianceBasicController
--- PASS: TestAccSiteBasicAwsResolverWithoutSecret (23.57s)
=== CONT  TestAccApplianceBasicControllerOverriderSPADisabled
--- PASS: TestAccMfaProviderBasic (23.61s)
=== CONT  TestAccApplianceBasicControllerWithoutOverrideSPA
--- PASS: TestAccRingfenceRuleBasicTCP (24.12s)
=== CONT  TestAccApplianceCustomizationBasic
--- PASS: TestAccDeviceScriptBasic (24.77s)
=== CONT  TestAccadministrativeRoleBasic
--- PASS: TestAccTrustedCertificateBasic (24.83s)
=== CONT  TestAccAppgateTrustedCertificateDataSource
--- PASS: TestAccEntitlementBasicPing (25.18s)
=== CONT  TestAccadministrativeMultiplePrivilegesValidation52
--- PASS: TestAccRadiusIdentityProviderBasic (25.47s)
=== CONT  TestAccAppgateMfaProviderDataSource
--- PASS: TestAccIPPoolBasic (25.77s)
=== CONT  TestAccadministrativeMultiplePrivilegesValidation
--- PASS: TestAccLdapCertificateIdentityProvidervBasic (25.82s)
=== CONT  TestAccAppgateApplianceCustomizationDataSource
--- PASS: TestAccApplianceBasicGateway (26.13s)
=== CONT  TestAccadministrativeRoleWithScope
--- PASS: TestAccLdapIdentityProviderBasic (26.23s)
=== CONT  TestAccEntitlementUpdateActionPortsSetOrder
=== CONT  TestAccadministrativeMultiplePrivilegesValidation52
    resource_appgate_administrative_role_test.go:264: Test is only for 5.2, privileges.target OnBoardedDevice
--- SKIP: TestAccadministrativeMultiplePrivilegesValidation52 (3.78s)
=== CONT  TestAccEntitlementUpdateActionHostOrder
--- PASS: TestAccAppgateTrustedCertificateDataSource (17.55s)
=== CONT  TestAccGlobalSettingsBasic
--- PASS: TestAccCriteriaScriptBasic (23.39s)
=== CONT  TestAccEntitlementUpdateActionOrder
--- PASS: TestAccAppgateMfaProviderDataSource (18.05s)
--- PASS: TestAccAppgateApplianceCustomizationDataSource (18.06s)
--- PASS: TestAccEntitlementBasicWithMonitor (44.58s)
--- PASS: TestAccConditionBasic (23.35s)
--- PASS: TestAccApplianceConnector (23.59s)
--- PASS: TestAccadministrativeRoleBasic (21.60s)
--- PASS: TestAccBlacklistUserBasic (24.31s)
--- PASS: TestAccadministrativeMultiplePrivilegesValidation (21.82s)
--- PASS: TestAccApplianceBasicControllerWithoutOverrideSPA (24.15s)
--- PASS: TestAccadministrativeRoleWithScope (21.84s)
--- PASS: TestAccApplianceBasicControllerOverriderSPADisabled (24.56s)
--- PASS: TestAccApplianceCustomizationBasic (29.46s)
--- PASS: TestAccSiteBasic (53.70s)
--- PASS: TestAccGlobalSettingsBasic (14.16s)
--- PASS: TestAccEntitlementUpdateActionPortsSetOrder (31.41s)
--- PASS: TestAccEntitlementUpdateActionHostOrder (29.22s)
--- PASS: TestAccSamlIdentityProviderBasic (58.53s)
--- PASS: TestAccEntitlementUpdateActionOrder (19.03s)
--- PASS: TestAccApplianceBasicController (39.33s)
--- PASS: TestAccClientProfileBasic (43.24s)
PASS
ok  	github.com/appgate/terraform-provider-appgatesdp/appgate	133.179s
```

</details>

acceptance test for `5.3.2-23587-release`
<details>

```
make testacc
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./appgate -v -count 1 -parallel 20  -timeout 120m
=== RUN   TestNewClient
--- PASS: TestNewClient (0.00s)
=== RUN   TestLoginInternalServerError
--- PASS: TestLoginInternalServerError (0.00s)
=== RUN   TestConfig
=== RUN   TestConfig/test_before_5.4
=== RUN   TestConfig/test_5.4_login
=== RUN   TestConfig/invalid_client_version
--- PASS: TestConfig (0.00s)
    --- PASS: TestConfig/test_before_5.4 (0.00s)
    --- PASS: TestConfig/test_5.4_login (0.00s)
    --- PASS: TestConfig/invalid_client_version (0.00s)
=== RUN   TestAccAppgateAdministrativeRoleDataSource
=== PAUSE TestAccAppgateAdministrativeRoleDataSource
=== RUN   TestAccAppgateApplianceCustomizationDataSource
=== PAUSE TestAccAppgateApplianceCustomizationDataSource
=== RUN   TestAccAppgateApplianceSeedDataSource
--- PASS: TestAccAppgateApplianceSeedDataSource (10.26s)
=== RUN   TestAccAppgateCADataSource
--- PASS: TestAccAppgateCADataSource (3.81s)
=== RUN   TestAccAppgateGlobalSettingsDataSource
--- PASS: TestAccAppgateGlobalSettingsDataSource (3.86s)
=== RUN   TestAccAppgateIdentityProviderDataSource
--- PASS: TestAccAppgateIdentityProviderDataSource (3.91s)
=== RUN   TestAccAppgateIPPoolDataSource
--- PASS: TestAccAppgateIPPoolDataSource (4.14s)
=== RUN   TestAccAppgateLocalUserDataSource
--- PASS: TestAccAppgateLocalUserDataSource (3.99s)
=== RUN   TestAccAppgateMfaProviderDataSource
=== PAUSE TestAccAppgateMfaProviderDataSource
=== RUN   TestAccAppgateTrustedCertificateDataSource
=== PAUSE TestAccAppgateTrustedCertificateDataSource
=== RUN   TestProvider
--- PASS: TestProvider (0.01s)
=== RUN   TestProvider_impl
--- PASS: TestProvider_impl (0.00s)
=== RUN   TestAccadministrativeRoleBasic
=== PAUSE TestAccadministrativeRoleBasic
=== RUN   TestAccadministrativeRoleWithScope
=== PAUSE TestAccadministrativeRoleWithScope
=== RUN   TestAccadministrativeMultiplePrivilegesValidation
=== PAUSE TestAccadministrativeMultiplePrivilegesValidation
=== RUN   TestAccadministrativeMultiplePrivilegesValidation52
=== PAUSE TestAccadministrativeMultiplePrivilegesValidation52
=== RUN   TestAccApplianceCustomizationBasic
=== PAUSE TestAccApplianceCustomizationBasic
=== RUN   TestAccApplianceBasicController
=== PAUSE TestAccApplianceBasicController
=== RUN   TestAccApplianceConnector
=== PAUSE TestAccApplianceConnector
=== RUN   TestAccApplianceBasicGateway
=== PAUSE TestAccApplianceBasicGateway
=== RUN   TestAccApplianceBasicControllerWithoutOverrideSPA
=== PAUSE TestAccApplianceBasicControllerWithoutOverrideSPA
=== RUN   TestAccApplianceBasicControllerOverriderSPADisabled
=== PAUSE TestAccApplianceBasicControllerOverriderSPADisabled
=== RUN   TestAccBlacklistUserBasic
=== PAUSE TestAccBlacklistUserBasic
=== RUN   TestAccClientConnectionsBasic
--- PASS: TestAccClientConnectionsBasic (4.83s)
=== RUN   TestAccClientProfileBasic
=== PAUSE TestAccClientProfileBasic
=== RUN   TestAccConditionBasic
=== PAUSE TestAccConditionBasic
=== RUN   TestAccCriteriaScriptBasic
=== PAUSE TestAccCriteriaScriptBasic
=== RUN   TestAccDeviceScriptBasic
=== PAUSE TestAccDeviceScriptBasic
=== RUN   TestAccEntitlementScriptBasic
=== PAUSE TestAccEntitlementScriptBasic
=== RUN   TestAccEntitlementBasicPing
=== PAUSE TestAccEntitlementBasicPing
=== RUN   TestAccEntitlementBasicWithMonitor
=== PAUSE TestAccEntitlementBasicWithMonitor
=== RUN   TestAccEntitlementUpdateActionOrder
=== PAUSE TestAccEntitlementUpdateActionOrder
=== RUN   TestAccEntitlementUpdateActionHostOrder
=== PAUSE TestAccEntitlementUpdateActionHostOrder
=== RUN   TestAccEntitlementUpdateActionPortsSetOrder
=== PAUSE TestAccEntitlementUpdateActionPortsSetOrder
=== RUN   TestAccGlobalSettingsBasic
=== PAUSE TestAccGlobalSettingsBasic
=== RUN   TestAccConnectorIdentityProviderBasic
--- PASS: TestAccConnectorIdentityProviderBasic (5.58s)
=== RUN   TestAccLdapCertificateIdentityProvidervBasic
=== PAUSE TestAccLdapCertificateIdentityProvidervBasic
=== RUN   TestAccLdapIdentityProviderBasic
=== PAUSE TestAccLdapIdentityProviderBasic
=== RUN   TestAccLocalDatabaseIdentityProviderBasic
--- PASS: TestAccLocalDatabaseIdentityProviderBasic (5.31s)
=== RUN   TestAccRadiusIdentityProviderBasic
=== PAUSE TestAccRadiusIdentityProviderBasic
=== RUN   TestAccSamlIdentityProviderBasic
=== PAUSE TestAccSamlIdentityProviderBasic
=== RUN   TestAccIPPoolBasic
=== PAUSE TestAccIPPoolBasic
=== RUN   TestAccLocalUserBasic
=== PAUSE TestAccLocalUserBasic
=== RUN   TestAccMfaProviderBasic
=== PAUSE TestAccMfaProviderBasic
=== RUN   TestAccAdminMfaSettingsBasic
=== PAUSE TestAccAdminMfaSettingsBasic
=== RUN   TestAccPolicyBasic
=== PAUSE TestAccPolicyBasic
=== RUN   TestAccRingfenceRuleBasicICMP
=== PAUSE TestAccRingfenceRuleBasicICMP
=== RUN   TestAccRingfenceRuleBasicTCP
=== PAUSE TestAccRingfenceRuleBasicTCP
=== RUN   TestAccSiteBasic
=== PAUSE TestAccSiteBasic
=== RUN   TestAccSiteBasicAwsResolverWithoutSecret
=== PAUSE TestAccSiteBasicAwsResolverWithoutSecret
=== RUN   TestAccTrustedCertificateBasic
=== PAUSE TestAccTrustedCertificateBasic
=== CONT  TestAccAppgateAdministrativeRoleDataSource
=== CONT  TestAccTrustedCertificateBasic
=== CONT  TestAccSiteBasicAwsResolverWithoutSecret
=== CONT  TestAccDeviceScriptBasic
=== CONT  TestAccApplianceBasicController
=== CONT  TestAccApplianceCustomizationBasic
=== CONT  TestAccSamlIdentityProviderBasic
=== CONT  TestAccadministrativeMultiplePrivilegesValidation52
=== CONT  TestAccSiteBasic
=== CONT  TestAccadministrativeMultiplePrivilegesValidation
=== CONT  TestAccRingfenceRuleBasicTCP
=== CONT  TestAccCriteriaScriptBasic
=== CONT  TestAccadministrativeRoleBasic
=== CONT  TestAccRingfenceRuleBasicICMP
=== CONT  TestAccEntitlementScriptBasic
=== CONT  TestAccConditionBasic
=== CONT  TestAccPolicyBasic
=== CONT  TestAccClientProfileBasic
=== CONT  TestAccAdminMfaSettingsBasic
=== CONT  TestAccadministrativeRoleWithScope
=== CONT  TestAccadministrativeMultiplePrivilegesValidation52
    resource_appgate_administrative_role_test.go:264: Test is only for 5.2, privileges.target OnBoardedDevice
--- SKIP: TestAccadministrativeMultiplePrivilegesValidation52 (3.91s)
=== CONT  TestAccRadiusIdentityProviderBasic
--- PASS: TestAccAppgateAdministrativeRoleDataSource (18.41s)
=== CONT  TestAccLdapIdentityProviderBasic
--- PASS: TestAccEntitlementScriptBasic (20.13s)
=== CONT  TestAccLdapCertificateIdentityProvidervBasic
--- PASS: TestAccCriteriaScriptBasic (20.81s)
=== CONT  TestAccGlobalSettingsBasic
--- PASS: TestAccadministrativeRoleBasic (21.27s)
=== CONT  TestAccEntitlementUpdateActionPortsSetOrder
--- PASS: TestAccRingfenceRuleBasicTCP (21.76s)
=== CONT  TestAccEntitlementUpdateActionHostOrder
--- PASS: TestAccTrustedCertificateBasic (22.00s)
=== CONT  TestAccEntitlementUpdateActionOrder
--- PASS: TestAccPolicyBasic (22.13s)
=== CONT  TestAccEntitlementBasicWithMonitor
--- PASS: TestAccDeviceScriptBasic (22.95s)
=== CONT  TestAccEntitlementBasicPing
--- PASS: TestAccadministrativeMultiplePrivilegesValidation (23.07s)
=== CONT  TestAccAppgateMfaProviderDataSource
--- PASS: TestAccConditionBasic (23.18s)
=== CONT  TestAccAppgateTrustedCertificateDataSource
--- PASS: TestAccSiteBasicAwsResolverWithoutSecret (23.31s)
=== CONT  TestAccLocalUserBasic
--- PASS: TestAccAdminMfaSettingsBasic (23.47s)
=== CONT  TestAccMfaProviderBasic
--- PASS: TestAccRingfenceRuleBasicICMP (23.90s)
=== CONT  TestAccIPPoolBasic
--- PASS: TestAccadministrativeRoleWithScope (23.99s)
=== CONT  TestAccAppgateApplianceCustomizationDataSource
--- PASS: TestAccRadiusIdentityProviderBasic (24.94s)
=== CONT  TestAccApplianceBasicControllerWithoutOverrideSPA
--- PASS: TestAccApplianceCustomizationBasic (38.63s)
=== CONT  TestAccBlacklistUserBasic
--- PASS: TestAccAppgateApplianceCustomizationDataSource (17.35s)
=== CONT  TestAccApplianceBasicControllerOverriderSPADisabled
--- PASS: TestAccAppgateTrustedCertificateDataSource (19.12s)
=== CONT  TestAccApplianceConnector
--- PASS: TestAccAppgateMfaProviderDataSource (19.29s)
=== CONT  TestAccApplianceBasicGateway
--- PASS: TestAccLdapCertificateIdentityProvidervBasic (24.58s)
--- PASS: TestAccLdapIdentityProviderBasic (26.71s)
--- PASS: TestAccGlobalSettingsBasic (24.53s)
--- PASS: TestAccEntitlementBasicPing (23.15s)
--- PASS: TestAccLocalUserBasic (23.53s)
--- PASS: TestAccMfaProviderBasic (24.37s)
--- PASS: TestAccIPPoolBasic (23.94s)
--- PASS: TestAccApplianceBasicControllerWithoutOverrideSPA (22.10s)
--- PASS: TestAccBlacklistUserBasic (15.86s)
--- PASS: TestAccSiteBasic (54.71s)
--- PASS: TestAccApplianceBasicController (55.65s)
--- PASS: TestAccEntitlementUpdateActionOrder (34.70s)
--- PASS: TestAccEntitlementUpdateActionPortsSetOrder (35.68s)
--- PASS: TestAccEntitlementUpdateActionHostOrder (35.48s)
--- PASS: TestAccEntitlementBasicWithMonitor (35.50s)
--- PASS: TestAccApplianceBasicGateway (15.37s)
--- PASS: TestAccApplianceConnector (15.46s)
--- PASS: TestAccApplianceBasicControllerOverriderSPADisabled (16.53s)
--- PASS: TestAccSamlIdentityProviderBasic (59.03s)
--- PASS: TestAccClientProfileBasic (62.46s)
PASS
ok  	github.com/appgate/terraform-provider-appgatesdp/appgate	108.196s
```

</details>

### CODE REVIEW

File: appgate/identity_provider.go [hunk 1]
```diff
@@ -350,15 +338,7 @@ func readProviderFromConfig(d *schema.ResourceData, provider openapi.IdentityPro
...
-	if v, ok := d.GetOk("display_name"); ok {
-		provider.SetDisplayName(v.(string))
-	}
-	if v, ok := d.GetOk("default"); ok {
-		provider.SetDefault(v.(bool))
-	}
-	if v, ok := d.GetOk("client_provider"); ok {
-		provider.SetClientProvider(v.(bool))
-	}
+
```
Comment (thomascellerier) [inline appgate/identity_provider.go:341]: Were those fields deprecated?

File: appgate/resource_appgate_appliance_test.go [hunk 2]
```diff
@@ -1606,8 +1594,8 @@ resource "appgatesdp_appliance" "test_controller" {
     https_port     = 444
     dtls_port      = 445
     allow_sources {
-      address = "1.3.3.7"
-      netmask = 0
+      address = "127.0.0.1"
+      netmask = 32
```
Comment (thomascellerier) [inline appgate/resource_appgate_appliance_test.go:1598]: Are all these changes due to stricter subnet validation on the controller side?

**Review by thomascellerier — APPROVED** [review]
Looks good.
It would be good to summarize what has been added/removed/deprecated though to get a better idea of the impact here.

File: appgate/identity_provider.go (hunk 1 above)
Comment (danijeel) [reply appgate/identity_provider.go:341]: yes from 5.1, I need to revisit the acceptance test for all identity providers to make them pass in 5.1, i will do it in another PR.

File: appgate/resource_appgate_appliance_test.go (hunk 2 above)
Comment (danijeel) [reply appgate/resource_appgate_appliance_test.go:1598]: yes :)

**Pull request merged**